lib/models/device.coffee

```diff
@@ -88,9 +88,32 @@ getDeviceModel = (deps, opts) ->
 		if semver.lt(version, minVersion)
 			throw new Error("Incompatible supervisor version: #{version} - must be >= #{minVersion}")
 
-	addApplicationName = (device) ->
+	###*
+	# @summary Get Dashboard device URL for a device
+	# @function getDashboardUrl
+	#
+	# @param {Object} options - options
+	# @param {String} options.appId - Application id
+	# @param {String} options.deviceId - Device id
+	#
+	# @returns {String} - Dashboard URL
+	# @throws Exception if either appId or deviceId are empty
+	#
+	# @example
+	# dashboardUrl = resin.models.device.getDashboardUrl({ appId: '012345', deviceId: '678901' })
```

Comment:
@pimterry Updated the comments and docs to use require `Number`, also added an `isFinite` check for the passed appId/deviceId options.

+missing changelog entry.